test/pools/pool_coarse_file.hpp

```diff
@@ -0,0 +1,19 @@
+// Copyright (C) 2024 Intel Corporation
+// Under the Apache License v2.0 with LLVM Exceptions. See LICENSE.TXT.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+#ifndef UMF_TEST_POOL_COARSE_FILE_HPP
+#define UMF_TEST_POOL_COARSE_FILE_HPP 1
+
+#include "umf/providers/provider_coarse.h"
+#include "umf/providers/provider_file_memory.h"
+
+#include "pool.hpp"
+#include "poolFixtures.hpp"
+
+using umf_test::test;
+using namespace umf_test;
+
+#define FILE_PATH ((char *)"/tmp/file_provider")
```

Comment:
why hard-coding this path? can't we use something like for dax (`UMF_TESTS_FSDAX_PATH`)?

Reply:
ops, race condition ;-)

Reply:
will remember

Reply:
Done: https://github.com/oneapi-src/unified-memory-framework/pull/821